projects/lendai-us/design-system/src/lib/badge/badge.component.scss

```diff
@@ -10,32 +10,32 @@ $radiuses: (
 
 $colors: (
   'blue': (
-    'background': c.color('blue', 100),
-    'foreground': c.color('blue', 700),
+    'background': c.color('info', 100),
```

Comment:
we talked so the badge will use prebuilt colors instead of going to color from `c.color` function